consensus/consensus-types/src/proof_of_store.rs

```diff
@@ -37,15 +37,23 @@ impl LogicalTime {
     Clone, Debug, Deserialize, Serialize, CryptoHasher, BCSCryptoHash, PartialEq, Eq, Hash,
 )]
 pub struct SignedDigestInfo {
+    pub batch_author: PeerId,
```

Comment:
I think that now we can remove PeerId from SingedDigest as it is redundant.

Reply:
As discussed offline, SignedDigest.peer_id is the signer, while this is the author

Reply:
I'll rename to SignedDigest.signer